src/main/java/org/hmhb/audit/DefaultAuditHelper.java

```diff
@@ -35,10 +50,20 @@ public String getCallerIp() {
         return request.getRemoteAddr();
     }
 
+    @Timed
     @Override
-    public String getCurrentUser() {
-        LOGGER.debug("getCurrentUser called");
-        return "someone";
+    public String getCurrentUserEmail() {
+        LOGGER.debug("getCurrentUserEmail called");
+
+        HmhbUser loggedInUser = authorizationService.getLoggedInUser();
+
+        String email = null;
+
+        if (loggedInUser != null) {
+            email = loggedInUser.getEmail();
```

Comment:
When adding authentication, I had forgotten to fill this in, so the audit data was still populating a hardcoded "someone".